bez3_description/urdf/bez3_simple.urdf

```diff
@@ -0,0 +1,1887 @@
+<?xml version="1.0" ?>
```

Comment:
what is the difference between this and bez3.urdf?

Reply:
this one has simple collision boxes. bez3 urdf uses stl for collision